site/app/views/course/CourseMaterialsView.php

```diff
@@ -15,14 +15,16 @@ class CourseMaterialsView extends AbstractView {
      * @param User[] $students
```

Comment:
Update the `@param` comment to relfect the change from `$user_group` to `$user` and get rid of the `return` param, thats outdated

Reply:
changed as above